frontend/src/components/Camera.jsx

```diff
@@ -1,7 +1,51 @@
 import Webcam from "react-webcam";
+import { useRef, useState, useCallback } from "react";
 
 function Camera() {
-  return <Webcam height={600} width={600} />;
+  const webcamRef = useRef(null);
+  const [imgSrc, setImgSrc] = useState(null);
+  const [lat, setLat] = useState();
+  const [long, setLong] = useState();
+
+  navigator.geolocation.getCurrentPosition((position) => {
+    setLat(position.coords.latitude);
+    setLong(position.coords.longitude);
+  });
```

Comment:
handle geolocation in an other PR, and try https://www.npmjs.com/package/react-geolocated